docs/maps/vector-style.asciidoc

```diff
@@ -86,12 +83,6 @@ Qualitative data driven styling is available for the following styling propertie
 * *Label color*
 * *Label border color*
 
-To ensure symbols are consistent as you pan, zoom, and filter the map, qualitative data driven styling uses a {ref}/search-aggregations-bucket-terms-aggregation.html[terms aggregation]. The term aggregation retrieves the top categories for the property. Feature values within the top categories are assigned a unique style. Feature values outside of the top categories are grouped into the *Other* category. A feature is assigned the *Other* category when the property value is undefined.
-
-To configure the terms aggregation, click the gear icon image:maps/images/gear_icon.png[]. Clear the *Get categories from indice* checkbox to turn off the terms aggregation request.
-
-NOTE: When the terms aggregation is not used, symbols might be inconsistent as users pan, zoom, and filter the map. Without terms aggregation, the top categories are calculated with data from the local layer. The top categories are re-calculated when layer data changes.
-
```

Comment:
I might've missed a relevant change, but why are we getting rid of this?

Reply:
All of this text was moved into the product as part of https://github.com/elastic/kibana/pull/84291. Not really needed any more since the UI really provides the context where its needed.